src/views/macros/svg-elements.html

```diff
@@ -1,7 +1,7 @@
 {% macro singleColourIcon(name) %}
 <svg class="icon">
     <title>{{name}}</title>
-    <use x="0" y="0" xlink:href="/assets/images/icons/{{name | lower}}.svg#icon"/>
+    <use x="0" y="0" xlink:href="/assets/images/icons/{{name | lower | replace(" ", "-") | replace(".", "")}}.svg#icon"/>
```

Comment:
Due to #390 we probably will need to separate the asset URL and name.  But that can probably be fixed in a separate pull.